homeassistant/components/fronius/strings.json

```diff
@@ -21,5 +21,219 @@
+      },
+      "power_factor_phase_2": {
+        "name": "[%key:component::sensor::entity_component::power_factor::name%] phase 2"
+      },
+      "power_factor_phase_3": {
+        "name": "[%key:component::sensor::entity_component::power_factor::name%] phase 3"
+      },
+      "power_factor": {
+        "name": "[%key:component::sensor::entity_component::power_factor::name%]"
+      },
+      "power_reactive_phase_1": {
+        "name": "[%key:component::sensor::entity_component::power::name%] reactive phase 1"
+      },
+      "power_reactive_phase_2": {
+        "name": "[%key:component::sensor::entity_component::power::name%] reactive phase 2"
+      },
+      "power_reactive_phase_3": {
+        "name": "[%key:component::sensor::entity_component::power::name%] reactive phase 3"
+      },
+      "power_reactive": {
+        "name": "[%key:component::sensor::entity_component::power::name%] reactive"
+      },
+      "power_real_phase_1": {
+        "name": "[%key:component::sensor::entity_component::power::name%] real phase 1"
+      },
+      "power_real_phase_2": {
+        "name": "[%key:component::sensor::entity_component::power::name%] real phase 2"
+      },
+      "power_real_phase_3": {
+        "name": "[%key:component::sensor::entity_component::power::name%] real phase 3"
+      },
+      "power_real": {
+        "name": "[%key:component::sensor::entity_component::power::name%] real"
+      },
+      "voltage_ac_phase_1": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 1"
+      },
+      "voltage_ac_phase_2": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 2"
+      },
+      "voltage_ac_phase_3": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 3"
+      },
+      "voltage_ac_phase_to_phase_12": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 1-2"
+      },
+      "voltage_ac_phase_to_phase_23": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 2-3"
+      },
+      "voltage_ac_phase_to_phase_31": {
+        "name": "[%key:component::fronius::entity::sensor::voltage_ac::name%] phase 3-1"
+      },
+      "energy_real_ac_consumed": {
+        "name": "[%key:component::sensor::entity_component::energy::name%] consumed"
+      },
+      "power_real_ac": {
+        "name": "[%key:component::sensor::entity_component::power::name%]"
+      },
+      "temperature_channel_1": {
+        "name": "[%key:component::sensor::entity_component::temperature::name%] channel 1"
+      },
+      "state_code": {
+        "name": "State code"
+      },
+      "state_message": {
+        "name": "State message"
+      },
+      "meter_mode": {
+        "name": "Meter mode"
+      },
+      "power_battery": {
+        "name": "[%key:component::sensor::entity_component::power::name%] battery"
```

Comment:
I did try
```json
"name": "[%key:component::sensor::entity_component::power::name%] [%key:component::sensor::entity_component::battery::name%]"
```
here - which worked ok with `python3 -m script.translations develop --integration fronius` but didn't pass pre-commit hassfest with error 
```
* [ERROR] [TRANSLATIONS] strings.json::entity::sensor::power_battery::name contains invalid reference component::sensor::entity_component::power::name%] [%key:component::sensor::entity_component::battery::name: Could not find name%]
```
Is using multiple links in one value not supported?

Reply:
That should not be done.
Only single key should be referenced, otherwise it will need to become new translations.

Reply:
do you mean using 2 references in one key?

Or should also this
```json
"name": "[%key:component::sensor::entity_component::power::name%] battery"
```
be changed to this:
```json
"name": "Power battery"
```

Reply:
Combining existing translations should not be done (it might not work for all languages).

So: `"name": "Power battery"`

Reply:
Alright, that leaves only 4 😬

I thought a translator doesn't have to use a referenced key in Lokalise - so it's optional. Would be pretty handy since we have a lot of "... pahse 1/2/3" or "... AC/DC" here and I guess that is the same for at least some languages.

Reply:
Shouldn't "Current AC phase 3" instead be "Phase 3 AC current"?
Shouldn't "Frequency AC" instead be "AC frequency"?
etc.

Reply:
I did initially name them that way, so users can see similar entities next to each other in alphabetically sorted lists. Eg
- Power AC
- Power AC Phase 1
- Power AC Phase 2
- Power AC Phase 3

Otherwise the distinct "Phase" entities would be separated from the cummulative "Power AC" entity.

Same for 
- Power real
- Power apparent

which a user might want to compare at a glance.

But I see the entity component does name it differently ("Apparent power"), so if you like I can change these.

Reply:
Feel free to wait for confirmation - it was just a suggestion as I am not 100% sure myself.
I just thought that "proper english" was expected in the translations.

Reply:
I would not take into accounting sorting in naming. That is resolving an UI issue in the backend (which is the wrong way around) :)

Reply:
Alright. I removed "AC" and "DC" wherever not necessary (eg. there is no apparent power in DC...) and moved the strings around 😃
Feel free to suggest changes, as I don't 100% know the english technical terms.